dist/stat_data.py

```diff
@@ -191,6 +191,7 @@ def __init__(self, name, desc, flags=''):
     # Cache and eviction statistics
     ##########################################
     CacheStat('cache_bytes_dirty', 'tracked dirty bytes in the cache', 'no_clear,no_scale,size'),
+    CacheStat('cache_bytes_dirty_total', 'bytes that have been dirtied in the cache', 'no_clear,no_scale,size'),
```

Comment:
"that have been dirtied in the cache" seems clunky, how about "bytes dirty in the cache cumulative" or just "bytes dirty cumulative" (since it's already a cache statistic)?

Reply:
FTR, I wish "tracked dirty bytes ..." had been "bytes dirty current" or similar to keep it sorted with related stats.